src/features/interactions.js

```diff
@@ -29,7 +29,7 @@ export function onMouseDown(event, ViewerDOM, tool, value, props, coords = null)
   switch (tool) {
     case TOOL_ZOOM_OUT:
       let SVGPoint = getSVGPoint(value, x, y);
-      nextValue = zoom(value, SVGPoint.x, SVGPoint.y, 1 / props.scaleFactor);
+      nextValue = zoom(value, SVGPoint.x, SVGPoint.y, 1 / props.scaleFactor, props);
```

Comment:
Originally I was passing props around for `scaleFactorMin` and `scaleFactorMax` values, but now they are being passed through the `value` argument.

I forgot to remove these props arguments in `zoom` and `stopZooming` functions. They should not be there.

Reply:
The point about this change is that I'm not sure that set `scaleFactorMin` and `scaleFactorMax` in the value object is the right way to handle this two props. These are two statics props and probably they don't change in the component lifecycle. Talking about refactor I think that the previous way to pass them was better.